cmd/rqlite/main.go

```diff
@@ -394,10 +396,13 @@ func getHTTPClient(argv *argT) (*http.Client, error) {
 		}
 	}
 
-	client := http.Client{Transport: &http.Transport{
-		TLSClientConfig: &tls.Config{InsecureSkipVerify: argv.Insecure, RootCAs: rootCAs},
-		Proxy:           http.ProxyFromEnvironment,
-	}}
+	client := http.Client{
+		Transport: &http.Transport{
+			TLSClientConfig: &tls.Config{InsecureSkipVerify: argv.Insecure, RootCAs: rootCAs},
+			Proxy:           http.ProxyFromEnvironment,
+		},
+		Timeout: httpTimeout,
```

Comment:
Could the timeout be configurable like the `insecure` option too? e.g `30s` or something that can be parsed with `time.ParseDuration`